source/scss/_library/_module.header.scss

```diff
@@ -4,6 +4,187 @@
 
 .c-main-header {
   text-align: center;
-  background-color: $c-secondary;
-  padding-top: $pad;
+  padding-top: 5px;
+
+  &__ad-unit {
+    position: relative;
+    z-index: 100;
+    background-color: $c-white;
+  }
+
+  &__inner {
+    background-color: $c-heading-bgcolor;
+    padding: $space-half $space-half $space-half 50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: relative;
+    height: 70px;
+
+    @include media('>xsmall') {
+      padding: $space-half $space $space-half 70px;
+    }
+  }
+
+  &__branding {
+    width: 120px;
```

Comment:
this seems brittle when thinking about other logos that may have to occupy this space. I'd prefer it if there was something like `&__branding > .gothamist-logo-svg { width: 120px; }`, so that we aren't stuck with a fixed width and a logo with an incompatible ratio.

Reply:
I agree. This should be part of a larger conversation about how these sites will be themed.